docs/installation.rst

```diff
@@ -18,7 +18,7 @@ The following HTTP libraries are supported:
 -  ``boto3``
 -  ``http.client``
 -  ``httplib2``
--  ``requests`` (both 1.x and 2.x versions)
+-  ``requests`` (2.x versions)
```

Comment:
Could elaborate on the intentions and the situation here?

Reply:
I changed this for update the docs, because we don't support requests 1.x and force to run docs validation.

The trigger to run docs validation are changes on docs folder.

```yaml
name: Validate docs

on:
  push:
    paths:
      - 'docs/**'
```

Reply:
@jairhenrique the trigger is clear but all I find regarding requests version support is the "vcrpy >=4.2.2 and requests <2.16.2 are not compatible" part. Either requests 1.x is supported or we should go further with blocking/detecting requests 1.x, e.g. in `setup.py`. Maybe a dedicated pull request would be the best fit?

Reply:
@hartwork I don't know how validation would be done in setup.py, since requests is not a project dependency.
As this doc points to the latest version and as of version 4.2.2 we no longer support versions < 2.16.2 via runtime, I see no problem keeping this change here.

Reply:
> As this doc points to the latest version and as of version 4.2.2 we no longer support versions < 2.16.2 via runtime, I see no problem keeping this change here.

@jairhenrique you're right, I mis-read "vcrpy >=4.2.2 and requests <2.16.2" earlier, my bad.

> @hartwork I don't know how validation would be done in setup.py, since requests is not a project dependency.

It's in `tests_require` in `setup.py`, we can change plain `requests` to `requests >=2.16.2` there.
Would you be up for adding this change to this pull request?

Reply:
Done!

Reply:
@jairhenrique thank you! :+1: